### PR TITLE
Focus 271, Create TaskWidget

### PR DESCRIPTION
**Pull request by rbadamsjr**

### CODE REVIEW

File: frontend/src/features/tasks/TaskItem.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+
+function TaskItem(props) {
```
Comment (jsnkle) [inline frontend/src/features/tasks/TaskItem.js:3]: let's get into the habit of de-structuring our props

File: frontend/src/features/tasks/TaskList.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import TaskItem from "./TaskItem";
+
+function TaskList(props) {
```
Comment (jsnkle) [inline frontend/src/features/tasks/TaskList.js:3]: Same here, de-structure

File: frontend/src/features/tasks/TasksWidget.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+import TaskList from "./TaskList";
+import { useGetMyTasksQuery, useUpdateTaskMutation } from "./tasksSlice";
+
```
Comment (jsnkle) [inline frontend/src/features/tasks/TasksWidget.js:3]: Shouldn't we make use of the Widget component? Maybe you didn't merge the develop branch into your branch to gain access to that and you're waiting?

**Review by jsnkle — COMMENTED** [review]
Looks good. Just had a few questions/comments.

File: frontend/src/features/tasks/TasksWidget.js (hunk 3 above)
Comment (rbadamsjr) [reply frontend/src/features/tasks/TasksWidget.js:4]: Fixed

File: frontend/src/features/tasks/TaskList.js (hunk 2 above)
Comment (rbadamsjr) [reply frontend/src/features/tasks/TaskList.js]: Fixed

File: frontend/src/features/tasks/TaskItem.js (hunk 1 above)
Comment (rbadamsjr) [reply frontend/src/features/tasks/TaskItem.js]: Fixed

**Comment by rbadamsjr** [comment]
See PR for branch FOCUS-271a

**Pull request closed without merging**